blocks/components/editable/index.js

```diff
@@ -10,6 +10,7 @@ import { Parser as HtmlToReactParser } from 'html-to-react';
  */
 import './style.scss';
 
+const KEYCODE_BACKSPACE = 8;
```

Comment:
At some point in the near future we may also consider leveraging the now-preferred [`event.key` property](http://caniuse.com/#feat=keyboardevent-key) which, funny enough, is better supported in IE back to IE11 than it is in Safari (only available in most recent versions).